languages/javascript/concepts/equality/about.md

```diff
@@ -0,0 +1,13 @@
+Javascript uses `==`, `===` and `Object.is()` to compare the equality in different instances
+
+In general as conclusion
+
+- If you know the type(s) in a comparison.If both of the types are same then `==` is identical to `===`.So prefering `==` will be shorter to write in such scenario
```

Comment:
Would these be better off as a bullet list? Also will need to check spacing between a period and the next word.